### DISCUSSION TITLE
How do we get ec number annotation with antismash

### DISCUSSION THREAD

**Discussion by Bio-finder** [antismash/antismash#676] [Q&A] [answered]
Hello,
I would like to predict secondary metabolism and thus I would need ec number annotation but for the moment I tried a few samples and didn't retrieve any ec number. Here is the command line that I've used:
`antismash $sample--taxon bacteria --output-dir $outputDir --cpus 5 --genefinding-tool prodigal 
  --fullhmmer --tigrfam --cc-mibig --cb-subclusters`
which command should I use to retrieve ec numbers?

**Comment by kblin** [comment, marked as answer]
antiSMASH does not provide EC number predictions. You'll need to use another tool for that.

**Comment by Bio-finder** [comment]
Ok, thank you, I thought that, at least in the past, it was doing that.
Do you know any tool that would give such prediction for secundary metabolites?

**Comment by kblin** [comment]
We did try to get useful EC predictions using `eficaz` back in the antiSMASH 3 days almost a decade ago. But if I recall correctly, predictions for enzymes specific to the secondary metabolism were pretty shaky and not really worth the added hours of runtime per genome and the hassle it was to install.

Back when we decided to drop `eficaz`, I remember I did try a couple of other prediction tools, but none of them actually predicted enough digits of the EC number reliably to be worth it. But I haven't looked at the EC prediction field in years, there might be some better tools out there now.

**Comment by Bio-finder** [comment]
Many thanks for the answer. I'll check this out!
Best regards,

**Comment by zreitz** [comment]
You can try [eggnog-mapper ](http://eggnog-mapper.embl.de/)as well. I've only used it for KEGG/GO, but it also gives ECs.

**Reply by Bio-finder** [reply]
Thank you. I have used it already and I wanted to find a way to annotate more by doing prediction as for secundary metabolism annotations aren't very exhaustive. I've bumped into `ECPred` and `deepEC` that I will try.
